src/org/traccar/sms/HttpSmsClient.java

```diff
@@ -38,13 +38,16 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(HttpSmsClient.class);
 
     private String url;
+    private String authorizationName;
```

Comment:
I think you should rename it to `authorizationHeader`.

Reply:
Renamed to `authorizationHeader`